challenge_03_loops-conditionals/loops-conditionals/conditionals/exercise_01.js

```diff
@@ -0,0 +1,16 @@
+/*
+    Create a program that starts with a number variable.
+    The program will return the variable is even or odd.
+
+    Change the variable to test it works for different cases.
+    Print result.
+
+    HINT: modulus operator ( % ) is your friend
+*/
+
+let number = 9;
```

Comment:
⚠️ FIX: use 🇬🇧, please.

⚠️ FIX:  change the variable to test it works for different cases. 

⚠️ FIX: add a comment to print expected output.